plugins/modules/ipatrust.py

```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Authors:
+#   Rob Verduijn <rob.verduijn@gmail.com>
+#
+# Copyright (C) 2019 By Me
+# see file 'COPYING' for use and warranty information
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import six
+from ansible.module_utils.ansible_freeipa_module import temp_kinit, \
+    temp_kdestroy, valid_creds, api_connect, api_command, date_format, \
+    compare_args_ipa, module_params_get, api_check_param, api_get_realm, \
+    api_command_no_name
+from ansible.module_utils._text import to_text
+from ansible.module_utils.basic import AnsibleModule
```

Comment:
Although PEP8 suggests that the imports are the firs items in the source code, Ansible modules have a different style to it. Start with ANSIBLE_METADATA, then Documentation, than the imports. Please, keep it consistent with other modules, despite of PEP8.

Reply:
done. 
( any way to explain this to the autoformatter in vscode ?)

Reply:
I  don't use vscode, so I can't help here.